lib/go/config/airy_yaml.go

```diff
@@ -32,3 +32,8 @@ type AiryConf struct {
 	Security   SecurityConf              `yaml:"security"`
 	Components map[string]ComponentsConf `yaml:"components,omitempty"`
 }
+
+type ConfigComponent struct {
+	Removed   bool   `json:"removed"`
+	Component string `json:"component"`
+}
```

Comment:
This is currently only used by the ConfigApplyResponsePayload, so not sure if it should be here or in `payloads`. Also `Removed` is just to be in line with the current concept (when a configuration is applied without a component - that component is removed).

I'd be happy to remove this and add a separate endpoint for removing a component.